resources/views/profile.blade.php

```diff
@@ -35,13 +42,34 @@
                         </div>
                     </div>
                     @if (Auth::user()->id !== $user->id)
-                        <div class="row justify-content-center mt-4">
-                            <div class="col">
-                                <button class="btn btn-primary btn-block">
-                                    Follow
-                                </button>
-                            </div>
-                        </div>
+                        @if ($relationship != null)
+                            <form action="/relationship/{{ $relationship->id }}" method="POST">
+                                @csrf
+                                @method('DELETE')
+                                <input type="hidden" name="follower_id" value="{{ Auth::user()->id }}">
```

Comment:
eliminate redundancies on form, and only o if condition on the method and button title

Reply:
I have resolved this in 03b4b95